numpy/__init__.py

```diff
@@ -132,6 +132,15 @@ class VisibleDeprecationWarning(UserWarning):
     pass
 
 
+class NoValue:
+    """Special keyword value.
+
+    This class may be used as the default value assigned to a
+    deprecated keyword in order to check if it has been given a user
+    defined value.
+    """
+    pass
+
```

Comment:
Can we not export this publicly? Just call it _NoValue or something? We
have enough public clutter as it is, and this adds zero value.
On Mar 14, 2015 9:15 PM, "Charles Harris" <notifications@github.com> wrote:

In numpy/__init__.py
<https://github.com/numpy/numpy/pull/5683#discussion_r26444409>:

> @@ -132,6 +132,15 @@ class VisibleDeprecationWarning(UserWarning):
>      pass
>
>
> +class NoValue:
> +    """Special keyword value.
> +
> +    This class may be used as the default value assigned to a
> +    deprecated keyword in order to check if it has been given a user
> +    defined value.
> +    """
> +    pass
> +

Two blank lines ;)

—
Reply to this email directly or view it on GitHub
<https://github.com/numpy/numpy/pull/5683/files#r26444409>.